src/jit/hwintrinsicxarch.cpp

```diff
@@ -1044,6 +1045,80 @@ GenTree* Compiler::impSSE2Intrinsic(NamedIntrinsic        intrinsic,
             break;
         }
 
+        case NI_SSE2_SetAllVector128:
```

Comment:
My plan is to have final refactoring of case `case NI_SSE2_SetAllVector128` after code gen problems will be solved.